timemachine/cpp/CMakeLists.txt

```diff
@@ -59,10 +75,10 @@ pybind11_add_module(${LIBRARY_NAME} SHARED NO_EXTRAS
   src/kernels/nonbonded_common.cu
 )
 
-include_directories(${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
 include_directories(src/kernels)
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/eigen)
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/${CUB_SRC_DIR})
+include_directories(SYSTEM ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
```

Comment:
Making these systems libraries suppresses any warnings within them, which is fine since we have no control over the code.